src/Microsoft.Data.Sqlite.Core/SqliteValueBinder.cs

```diff
@@ -64,18 +71,44 @@ public virtual void Bind()
             }
             else if (type == typeof(char))
             {
-                var value = (long)(char)_value;
-                BindInt64(value);
+                var chr = (char)_value;
+                if (_sqliteType == SqliteType.Text)
+                {
+                    BindText(new string(chr, 1));
+                }
+                else
+                {
+                    var value = (long)chr;
```

Comment:
Should this value be inlined? If no then new string should be saved in value too.

Reply:
Done.